server/routes/api/v1/v1.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const campaigns = require('./campaigns') // If the routes file is named index.js, we can omit it when we do the import instead of writing './campaigns/index'.
+// const { exampleMiddleware } = require('../../../middleware/example')
```

Comment:
Can you please delete the extra comments and any commented out code?

Reply:
Yes, I have now taken them out